### DISCUSSION TITLE
can't run script with poetry

### DISCUSSION THREAD

**Discussion by yoan34** [python-poetry/poetry#9199] [Q&A] [answered]
Hello, I try to run a script with tool.poetry.scripts but it doesn't work, i list the step that I made:
File structure:
📁 backend/
...
├── 📁 wia/
│       └─ 📄 _ _init_ _.py
│       └─ 📄 main.py # an app inside
└── 📄 pyproject.toml

And the contain of pyproject.toml is:
```
[tool.poetry]
name = "backend"
version = "0.1.0"
description = ""
authors = ["ybousquet <**********@gmail.com>"]
readme = "README.md"

[tool.poetry.scripts]
wia = "wia.main:app"

[tool.poetry.dependencies]
python = "^3.11"


[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"
```
The step i follow is:
 - poetry init
 - poetry shell (create and active virtual env)
 - poetry install

After that, when i run the command `poetry run wia` i have the output:
```
Warning: 'wia' is an entry point defined in pyproject.toml, but it's not installed as a script. You may get improper `sys.argv[0]`.

The support to run uninstalled scripts will be removed in a future release.

Run `poetry install` to resolve and get rid of this message.


No file/folder found for package backend
```
And when i run `poetry install` i have the ouput:
```
Installing dependencies from lock file

Installing the current project: backend (0.1.0)
The current project could not be installed: No file/folder found for package backend
If you do not want to install the current project use --no-root
```

**Comment by abn** [comment, marked as answer]
You are likely missing package configuration. See https://python-poetry.org/docs/pyproject/#packages.

Since your project is called `backend` and no package named `backend` exists, it is not installed. 

```toml
[tool.poetry]
# ...
packages = [
    { include = "wia" },
]
```

**Comment by yoan34** [comment]
Oh thanks you, it's work perfectly
